src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DoubleInput.cs

```diff
@@ -28,7 +28,7 @@ public void CustomizeView(DoubleInput nodeModel, NodeView nodeView)
 
             tb.DataContext = nodeModel;
 
-            tb.BindToProperty(new Binding("Value")
+            tb.BindToProperty(new Binding("NumericalValue")
             {
                 Mode = BindingMode.TwoWay,
                 Converter = new DoubleInputDisplay(),
```

Comment:
Also, if we're limiting the ability for the user to put a range into the `DoubleInput` node, then any backing property we add should just have the type we want to store. Like `OurNewValue : double`. And that is a further reason to use a converter because a string will come in from the text box, and a double will be stored.